workflow/tropical-cyclone/STORM.smk

```diff
@@ -195,7 +202,7 @@ rule wrap_storm_fixed:
         temp("{OUTPUT_DIR}/input/STORM/fixed/{STORM_MODEL}/{STORM_BASIN}/STORM_FIXED_RETURN_PERIODS_{STORM_MODEL}_{STORM_BASIN}_{STORM_RP}_YR_RP.wrapped.tif")
     shell:
         """
-        gdalwarp -te -180 -60 180 60 {input} {output}
+        gdalwarp -te -179.85 -60.15 180.15 59.95 -co COMPRESS=LZW -co PREDICTOR=2 -co TILED=YES {input} {output}
```

Comment:
Any particular reason why the changes are -0.15, 0.15, 0.15 and -0.05? I'd naively expect them to be the same in magnitude